source/concepts/applications.rst

```diff
@@ -4,99 +4,91 @@ Applications
+  server space. For example, in VR the coordinate system may be fixed in physical
+  space such that it can't be changed directly to match server space. The server
+  is not aware of this and it is the client's responsibility to transform
+  coordinates into server space before communicating them.
 
-The client is free to represent the server space anywhere in its game space.
+The client is free to represent the server space anywhere in its client space.
 Optionally, the server can suggest to a client where to place and how to orient
-the game space relative to the server space with the :ref:`user-origin
+the client space relative to the server space with the :ref:`user-origin
 <user-origin-description>` key.
 
+.. note::
+
+   In MD applications there is an additional **simulation space** coordinate
+   system. Its relation to the server space is determined by the position
+   of the simulation box (via `scene` key) and iMD interactions use this
+   coordinate space.
+
+
 .. _avatar-description:
 
 Avatars
 ~~~~~~~
 
-A client can expose an avatar to the other clients by periodically updating the
-corresponding key in the :ref:`shared state <state-updates>`. Since client get
-updates 30 times per seconds by default, 30 avatar updates per seconds is the
-recommended update rate.
+Users may share their presence in the virtual space by creating and continuously
+updating an "avatar". For example, in the iMD-VR application, each VR client
+shares their head and hand positions for others to see.
 
-Avatars are represented in the shared state as a protobuf `Struct
+Avatars are exchanged via the shared state as a protobuf `Struct
 <https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Struct>`_
-under a key formatted as ``avatar.<PLAYER_ID>``.
-
-Inside the Struct, an avatar refers to its player ID. It can also specify a
-name and a color. The most important part of an avatar is the list of its
-components. Components are elements of the avatar with a position and a rotation.
+under keys of the form ``avatar.<PLAYER_ID>``. This avatar Struct details the
+user's player ID, a name for the user, a display color, and the list of its
+spatial components.
 
 .. warning::
 
    The avatar specifies its player ID twice: in the top level key and with the
-   ``playerid`` sub-key. Both places MUST match. A discrepancy between the two
-   is undefined behaviour. See `issue #98 in nanover-server-py
+   ``playerid`` sub-key. Both places MUST match. See `issue #98 in nanover-server-py
    <https://github.com/IRL2/nanover-server-py/issues/98>`_.
 
-The name is an arbitrary string meant to be displayed to the other users as a
-human-readable identifier. The color is also meant to be displayed to the other
-users. The color is provided as a list of RGBA values between 0 and 1.
+The name is an arbitrary string typically displayed as a name tag to the other
```

Comment:
What does "name tag" mean here exactly? Should this say something like: "The name is an arbitrary string that is displayed on the headset of the player's avatar, which is visible to other players in VR."